components/brave_rewards/resources/tip/components/app.tsx

```diff
@@ -28,18 +29,20 @@ export class App extends React.Component<Props, {}> {
     return this.props.actions
   }
 
-  getTipBanner = (url: string, publisher: RewardsTip.Publisher, mediaMetaData: RewardsTip.MediaMetaData | undefined) => {
+  getTipBanner = (url: string, monthly: boolean, publisher: RewardsTip.Publisher, mediaMetaData: RewardsTip.MediaMetaData | undefined) => {
```

Comment:
maybe instead of passing arg in, we can just access props directly in this func? wdyt?

Reply:
fine by me, done